tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -4926,6 +4925,34 @@ def test_search_logged_models_filter_string(store: SqlAlchemyStore):
     assert [m.name for m in models] == [model_1.name]
     assert models.token is None
 
+    for val in (
+        # A single item without a comma
+        f"('{model_1.name}')",
+        # A single item with a comma
+        f"('{model_1.name}',)",
+        # Multiple items
+        f"('{model_1.name}', 'foo')",
+    ):
+        # IN
+        models = store.search_logged_models(
+            experiment_ids=[exp_id_1],
+            filter_string=f"name IN {val}",
+        )
+        assert [m.name for m in models] == [model_1.name]
+        assert models.token is None
+        # NOT IN
+        models = store.search_logged_models(
+            experiment_ids=[exp_id_1],
+            filter_string=f"name NOT IN {val}",
+        )
+        assert [m.name for m in models] == []
+
+    models = store.search_logged_models(
+        experiment_ids=[exp_id_1],
+        filter_string=f"name NOT IN ('{model_1.name}', 'foo')",
+    )
+    assert [m.name for m in models] == []
```

Comment:
```suggestion
```

Reply:
This test is to make sure multiple items are parsed fine.

Reply:
But you have it tested in line 4934?

Reply:
ah you're right!